cassandra-trunk/Dockerfile-trunk.ubi8

```diff
@@ -40,6 +40,7 @@ ARG COMMITSHA="HEAD"
 ARG CASSANDRA_BRANCH="trunk"
 ENV CASSANDRA_PATH=/opt/cassandra
 ENV CASSANDRA_FILES_PATH=/opt/cassandra_files
+RUN apt-get update && apt-get install -y python3
```

Comment:
:suggestion

Can we simply move this to line 49 below and include it with the list of things being installed there? So that line would look like:
```
&& apt-get install -y --no-install-recommends git ant ant-optional make python3 \
```

Reply:
oh I totally missed this, sure, good one.